atmo/clusters/models.py

```diff
@@ -248,7 +251,7 @@ def is_ready(self):
     @property
     def is_expiring_soon(self):
         """Returns true if the cluster is expiring in the next hour."""
-        return self.end_date <= timezone.now() + timedelta(hours=1)
+        return self.end_date <= timezone.now() + timedelta(hours=self.lifetime)
```

Comment:
If you change this I think the name is wrong. `is_expiring_soon` wouldn't make sense if `lifetime` was 8 hours away. That doesn't seem "soon" to me.